pyproject.toml

```diff
@@ -6,8 +6,12 @@ requires = [
     "setuptools>=34.4",
 
     # Plugins
-    "setuptools_scm[toml]>=3.5",
-    "setuptools_scm_git_archive>=1.1",
+    "setuptools-scm >= 3.5; python_version >= '3.7'",
+    #  ^    supports git archives through a plugin
+    #  |                                        |
+    # supports git archives natively            V
+    "setuptools-scm[toml] >= 3.5; python_version < '3.7'",
```

Comment:
This must be capped below v7 so that it's only paired with the plugin when git archives are not supported natively.